web/modules/custom/usagov_wizard/usagov_wizard.module

```diff
@@ -412,7 +412,9 @@ function usagov_wizard_get_term_breadcrumb(Term $rootTerm): array {
       if (isset($crumb['uri'])) {
         $crumb['url'] = Url::fromUri($crumb['uri'])->toString();
         // Sometimes Drupal doesn't prepend Spanish path prefix.
-        if ($rootTerm->language()->getId() === 'es' && !str_starts_with($crumb['url'], '/es/')) {
+        // Sometimes this crumb's uri is "route:<nolink>" and the url is an empty string;
+        // in that case we should leave it empty (current page, not to be linked).
+        if ($crumb['url'] && $rootTerm->language()->getId() === 'es' && !str_starts_with($crumb['url'], '/es/')) {
```

Comment:
This works and I'll approve it. I've learned a bit about Url object in the last 3.5 weeks and Drupal supports getting the prefixed URL directly by changing line 413 to the code below. We can then remove the `if` clause, Drupal return an empty string if it can't find the alias for the URI. Also explains why only the Spanish pages were affected.

```php
$crumb['url'] = Url::fromUri($crumb['uri'], ['language' => $rootTerm->language()])->toString();
```

I tested that locally for the Spanish pages and the breadcrumbs are correct but I'd want to do the whole tome export, compare HTML against a known-good export, to make sure there aren't unintended consequences.